deploy/deploy.sh

```diff
@@ -45,6 +47,15 @@ cd "$HTDOCS/wp-content/"
 rm -rf uploads
 ln -s ../../../uploads uploads
 
+# Setup mu-plugins if VIP
+VIP=$(cat "$hosts_file" | shyaml get-value "$CI_SCRIPT_OPTIONS.vip" | tr '[:upper:]' '[:lower:]')
+if [ "$VIP" = "true" ]; then
+    MU_PLUGINS_URL=${MU_PLUGINS_URL:-"https://github.com/Automattic/vip-mu-plugins-public"}
+    MU_PLUGINS_DIR="$HTDOCS/wp-content/mu-plugins"
+    echo "Cloning mu-plugins from: $MU_PLUGINS_URL"
+    git clone -q --recursive "$MU_PLUGINS_URL" "$MU_PLUGINS_DIR"
```

Comment:
Let's `--depth=1` since we will be re-deploying it everytime so we don't need previous commits? @mrrobot47

Reply:
Yes, the previous commits are not required at all. Great suggestion! Have updated the PR.